exec/vector/src/main/java/org/apache/drill/exec/expr/fn/impl/DateUtility.java

```diff
@@ -639,29 +648,95 @@ public static String getTimeZone(int index) {
     return timezoneList[index];
   }
 
+  /**
```

Comment:
The "parseLocalDate", "parseLocalTime", "parseLocalDateTime" are used by various junit tests.  These parsers are strict in that if the input string doesn't have all the specified fields, it will fail to parse.